src/misc.c

```diff
@@ -282,6 +285,54 @@ static JSValue tjs_print(JSContext *ctx, JSValueConst this_val, int argc, JSValu
     return JS_UNDEFINED;
 }
 
+#define STR_BUFFER_SIZE 128
+#define STR_BUFFER_INCREMENT 16
+
+static JSValue tjs_prompt(JSContext *ctx, JSValueConst this_val, int argc, JSValueConst *argv) {
```

Comment:
I think it would better to just use 1 argument here and do the concatenation into a single string in JS. That would save us from all the realloc-ing.

Reply:
Turns out ```window.prompt``` is actually ```window.prompt([message[, defaultValue]])``` according to https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt so I implemented that.